### PR TITLE
fix: open reporting in new tab full screen

### LINKED ISSUE #13658: [CAL-3123] open routing forms reporting in new tab
this is too crowded for a reporting table:

CleanShot 2024-02-13 at 14 58 20@2x

instead lets open this in full screen as a new tab

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-3123](https://linear.app/calcom/issue/CAL-3123/open-routing-forms-reporting-in-new-tab)</sub>

### PR DESCRIPTION
**Pull request by PeerRich**
CleanShot 2024-02-13 at 15 17 33@2x

### CODE REVIEW

**Review by Udit-takkar — COMMENTED** [review]
https://github.com/calcom/cal.com/assets/53316345/1482dfe5-2824-4f30-b531-2c49468ed835

1) can't see some elements when banner is active
2) hydration err

Screenshot 2024-02-14 at 1 46 11 PM

**Review by hariombalhara — DISMISSED** [review]
Hydration error is pre-existing. We can merge it when we fix the banner issue.

-- Edit 
There is a test failure, fixing that.

**Review by CarinaWolli — CHANGES_REQUESTED** [review]
Banner on top and mobile view needs to be fixed

Mobile:
Screenshot 2024-02-21 at 2 24 02 PM

File: packages/app-store/routing-forms/pages/reporting/[...appPages].tsx [hunk 1]
```diff
@@ -166,18 +166,16 @@ const Reporter = ({ form }: { form: inferSSRProps<typeof getServerSideProps>["fo
...
-          {...config}
-          value={query.state.tree}
-          onChange={(immutableTree, config) => {
-            onChange(immutableTree, config as QueryBuilderUpdatedConfig);
-          }}
-          renderBuilder={renderBuilder}
-        />
-        <Result formId={form.id} jsonLogicQuery={jsonLogicQuery as JsonLogicQuery} />
-      </div>
+    <div className="cal-query-builder bg-default fixed inset-0 w-full overflow-scroll ltr:mr-2 rtl:ml-2">
```
Comment (PeerRich) [inline packages/app-store/routing-forms/pages/reporting/[...appPages].tsx:169]:
```suggestion
    <div className="cal-query-builder bg-default fixed inset-0 w-full overflow-scroll pt-12 ltr:mr-2 rtl:ml-2 sm:pt-0">
```

**Review by CarinaWolli — APPROVED** [review]

**Pull request merged**